### PR TITLE
Added Calvin's favorite song

### PR DESCRIPTION
**Pull request by armstrca**

### CODE REVIEW

**Comment by armstrca** [comment]
Oh brutal we changed the formatting completely while I was doing that 😂

File: button.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.btn-liquid {
```
Comment (heratyian) [inline button.css:1]: let's keep it simple with bootstrap. no custom styles.

File: button.js [hunk 2]
```diff
@@ -0,0 +1,279 @@
+$(function() {
```
Comment (heratyian) [inline button.js:1]: no need for custom js

File: index.html [hunk 3]
```diff
@@ -1,70 +1,69 @@
...
+        <td>
+          <a href="https://shed-official.bandcamp.com/track/the-shadow-of-totality-2" class="btn-liquid">
+            <span class="inner">Link</span>
+          </a>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</body>
+</html>
```
Comment (heratyian) [inline index.html:69]: newline at end of file

**Review by heratyian — CHANGES_REQUESTED** [review]
Might need to pull latest and re-submit

**Pull request closed without merging**